header-footer-grid/Core/Builder/Header.php

```diff
@@ -94,7 +94,7 @@ public function init() {
 					),
 				),
 				'builderMigrated' => $migrated_hfg,
-				'hadOldBuilder'   => neve_had_old_hfb() && ! neve_is_new_skin(),
+				'hadOldBuilder'   => neve_had_old_hfb() && ( ! neve_is_new_skin() || neve_was_auto_migrated_to_new() ),
```

Comment:
This allows the migration for the builder to be displayed if `old_skin` or if we `auto_migrated` since if we do the auto migration we need to allow them to switch back even if on new.